Makefile

```diff
@@ -171,7 +171,6 @@ test-symbolicator:
 
 test-acceptance: node-version-check
 	sentry init
-	make build-platform-assets
```

Comment:
Is there more that can be removed around this?

Reply:
I tried but there's some extra code on python side which I'm not sure if it is safe to remove: https://github.com/getsentry/sentry/blob/55237a079f40b339d8dfe3d7b1f538bf2054a4d5/src/sentry/utils/integrationdocs.py#L117-L132

Reply:
Acceptance tests failed without this as BE relies on these paths existing. Resolving for now.